vmck/settings.py

```diff
@@ -1,7 +1,7 @@
 import os
 import sentry_sdk
 from .utils import is_true
-from .base_settings import *
```

Comment:
Leave (and ignore) this star import, we wouldn't want to update this list too on every change in that file.